js/mercado.js

```diff
@@ -213,18 +213,11 @@ module.exports = class mercado extends Exchange {
         const currencyIds = Object.keys (balances);
         for (let i = 0; i < currencyIds.length; i++) {
             const currencyId = currencyIds[i];
-            let code = currencyId;
-            if (currencyId in this.currencies_by_id) {
-                code = this.currencies_by_id[currencyId]['code'];
-            } else {
-                code = this.commonCurrencyCode (currencyId.toUpperCase ());
-            }
-            // const currencyId = this.currencyId (code);
-            const lowercase = currencyId.toLowerCase ();
```

Comment:
We should check this, because, I think, their balance has lowercase currency ids, whereas all other endpoints, including the markets endpoints return the ids in uppercase.

Reply:
Ahh, sorry, missed `if (currencyId in balances)`, nevermind.